common/src/utils/git.ts

```diff
@@ -0,0 +1,33 @@
+import { exec } from "child_process";
+import { promisify } from "util";
+
+const universalExec: typeof exec =
```

Comment:
I had to extract git utils eventually and make a fast-to-implement shim for the browser.

Reply:
Moved it to a separate commit: https://github.com/near/near-explorer/commit/c534ea34dc41b464963aca44d7ff06572e643f5a